encoding/rs/frame_coeffs.go

```diff
@@ -0,0 +1,112 @@
+package rs
+
+import (
+	"encoding/binary"
+	"fmt"
+	"github.com/Layr-Labs/eigenda/encoding"
+	"github.com/consensys/gnark-crypto/ecc/bn254/fr"
+)
+
+// FrameCoeffs is a slice of coefficients (i.e. an encoding.Frame object without the proofs).
```

Comment:
Where is "encoding.Frame" defined now?

Reply:
`encoding.Frame` is different than `rs.Frame` (renamed in this PR to `rs.FrameCoeffs`). The old naming pattern was really confusing... these objects had the same name but different data. The location where `encoding.Frame` is defined did not change. All things that used the old `rs.Frame` have been renamed to use `rs.FrameCoeffs`.